js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -443,7 +455,7 @@ export default class HowTo extends Component {
 			<div className={ classNames }>
 				{ ( hasDuration && typeof timeString === "string" && timeString.length > 0 ) &&
 					<p className="schema-how-to-total-time">
-						{ __( "Time needed:", "wordpress-seo" ) }
+						{ durationText }
 						&nbsp;
```

Comment:
since we now allow the timeString to wrap in a new line, I think this `&nbsp;` should be removed